src/GeositeFramework/js/Map.js

```diff
@@ -33,14 +36,10 @@ require(['use!Geosite',
     'use strict';
 
     function getSelectedBasemapLayer(model, esriMap) {
-        // Return an ESRI layer object for the currently-selected basemap spec
+        // Add the selected basemap to the Esri map
         var basemap = getSelectedBasemap(model);
-        if (basemap.layer === undefined) {
-            // This basemap has no layer yet, so make one and cache it
-            basemap.layer = new TileLayer(basemap.url);
-            esriMap.addLayer(basemap.layer);
-        }
-        return basemap.layer;
+        basemap.layer = new TileLayer(basemap.url);
```

Comment:
Is there a reason to dispose of the mechanism to cache the layer creation, it seems like the only substantial change here is to change `esriMap.addLayer` to `esriMap.add`, or is there more going on?